examples/all-clusters-app/nrfconnect/main/AppTask.cpp

```diff
@@ -191,12 +191,20 @@ CHIP_ERROR AppTask::Init()
     SetDeviceInstanceInfoProvider(&mFactoryDataProvider);
     SetDeviceAttestationCredentialsProvider(&mFactoryDataProvider);
     SetCommissionableDataProvider(&mFactoryDataProvider);
+    // Read EnableKey from the factory data.
+    MutableByteSpan enableKey(sTestEventTriggerEnableKey);
+    err = mFactoryDataProvider.GetEnableKey(enableKey);
```

Comment:
Here and in any other places of using GetEnableKey. Please consider adding 
#if CONFIG_CHIP_DEVICE_ENABLE_KEY check.